<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTMainWindow</class>
 <widget class="QMainWindow" name="PTMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="leftVerticalLayout">
      <item>
       <layout class="QHBoxLayout" name="topLeftHorizontalLayout"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightVerticalLayout">
      <item>
       <layout class="QHBoxLayout" name="topRightHorizontalLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExecute">
    <property name="title">
     <string>Execute</string>
    </property>
    <addaction name="actionCalculateFotoTri"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuExecute"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSet_Mark"/>
   <addaction name="actionMove"/>
   <addaction name="actionZoom"/>
   <addaction name="actionFit_View"/>
   <addaction name="actionFlightDirection"/>
   <addaction name="actionFoto_Tri"/>
   <addaction name="actionView_Report"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="leftImageDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Left Image Points</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="leftImageComboBox"/>
     </item>
     <item>
      <widget class="ETableWidget" name="leftImageTableWidget">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Table with image's points</string>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <column>
        <property name="text">
         <string>Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Col</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Lin</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>key</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="pointsDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>All Registered Points</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;All Points&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ETableWidget" name="pointsTableWidget">
       <property name="toolTip">
        <string>Table with all points</string>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>6</number>
       </property>
       <column>
        <property name="text">
         <string>Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>E</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>N</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>H</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>keys</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="rightImageDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Right Image Points</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QComboBox" name="rightImageComboBox"/>
     </item>
     <item>
      <widget class="ETableWidget" name="rightImageTableWidget">
       <property name="toolTip">
        <string>Table with image's points</string>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <column>
        <property name="text">
         <string>Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Col</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Lin</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Key</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSet_Mark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/mark.png</normaloff>:/icon/mark.png</iconset>
   </property>
   <property name="text">
    <string>actionSet_Mark</string>
   </property>
   <property name="toolTip">
    <string>Mark point in image</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/move.png</normaloff>:/icon/move.png</iconset>
   </property>
   <property name="text">
    <string>move</string>
   </property>
   <property name="toolTip">
    <string>Move a image</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/find.png</normaloff>:/icon/find.png</iconset>
   </property>
   <property name="text">
    <string>zoom</string>
   </property>
   <property name="toolTip">
    <string>Zoom in image</string>
   </property>
  </action>
  <action name="actionFit_View">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/fit.png</normaloff>:/icon/fit.png</iconset>
   </property>
   <property name="text">
    <string>fit</string>
   </property>
   <property name="toolTip">
    <string>Fit image in all possible area</string>
   </property>
  </action>
  <action name="actionFoto_Tri">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/exterior.png</normaloff>:/icon/exterior.png</iconset>
   </property>
   <property name="text">
    <string>fotoTri</string>
   </property>
   <property name="toolTip">
    <string>Run PhotoTriangulation</string>
   </property>
  </action>
  <action name="actionView_Report">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/table.png</normaloff>:/icon/table.png</iconset>
   </property>
   <property name="text">
    <string>viewReport</string>
   </property>
   <property name="toolTip">
    <string>See the results/report</string>
   </property>
  </action>
  <action name="actionFlightDirection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/fd.png</normaloff>:/icon/fd.png</iconset>
   </property>
   <property name="text">
    <string>flightdirection</string>
   </property>
   <property name="toolTip">
    <string>Indicates flight's direction</string>
   </property>
  </action>
  <action name="actionCalculateFotoTri">
   <property name="text">
    <string>Calculate</string>
   </property>
   <property name="toolTip">
    <string>Calculate PhotoTriangulation</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ETableWidget</class>
   <extends>QTableWidget</extends>
   <header>ETableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
